scripts/migrate-files-to-s3.py

```diff
@@ -0,0 +1,92 @@
+# pylint: disable=invalid-name
+import io
+from server.models import *  # pylint: disable=wildcard-import
+from server.database import db_session
+from server.util.file import store_file, timestamp_filename
+
+
+def migrate_file(file, path):
+    if not file.storage_path:
+        storage_path = store_file(io.BytesIO(file.contents.encode("utf-8")), path)
```

Comment:
if this fails, can you just re-run the script? I think so from reading it, just checking.

Reply:
yep, it's built to be re-run on failures. one transaction per audit